Octokit.Tests.Integration/Clients/PullRequestsReviewRequestClientTests.cs

```diff
@@ -0,0 +1,164 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Octokit;
+using Octokit.Tests.Integration;
+using Octokit.Tests.Integration.Helpers;
+using Xunit;
+
+public class PullRequestsReviewRequestClientTests : IDisposable
+{
+    private readonly IGitHubClient _github;
+    private readonly IPullRequestReviewRequestsClient _client;
+    private readonly RepositoryContext _context;
+
+    const string branchName = "new-branch";
+    const string collaboratorLogin = "m-zuber-octokit-integration-tests";
```

Comment:
I'll change it :) But in my defense: I've take it from `RepositoryCollaboratorClientTests.cs` and ran the test.